.php-cs-fixer.dist.php

```diff
@@ -4,7 +4,11 @@
 
 $finder = (new PhpCsFixer\Finder())
     ->in(__DIR__)
-    ->exclude('var')
+    ->exclude([
+        'var',
+        'node_modules',
+        'vendor',
```

Comment:
Thanks for this, small suggestion

```suggestion
        'vendor',
        'docker',
```

of adding docker to this too. I could never run it and get

```
/MbinOrg/mbin/docker/storage/postgres): Failed to open directory: Permission denied
```

So I had been running it by specifying the php files I changed individually each time, should have actually looked to see that this was possible :sweat_smile: